src/main/java/com/minecolonies/coremod/entity/ai/citizen/deliveryman/EntityAIWorkDeliveryman.java

```diff
@@ -435,13 +436,13 @@ private boolean hasTools(@NotNull final AbstractBuilding buildingToDeliver)
             return true;
         }
 
-        if (requiredTool.equals(Utils.PICKAXE))
+        /*if (requiredTool.equals(Utils.PICKAXE))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)